test/Swashbuckle.AspNetCore.TestSupport/Fixtures/TypeWithRequiredProperty.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Swashbuckle.AspNetCore.TestSupport
+{
+    public class TypeWithRequiredProperty
```

Comment:
I would suggest making this a nested type inside the test class using it instead, and then wrapping the whole thing in the `#if`, rather than us always compiling an empty class that isn't used for other targets.